docs/conf.py

```diff
@@ -62,3 +66,24 @@
     "collapse_navigation": False,
     "prev_next_buttons_location": "None",
 }
+
+
+def linkcode_resolve(domain, info):
```

Comment:
Can you say a few words for what this function is used for?

Reply:
Done!